src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStoreV1.swift

```diff
@@ -50,10 +50,48 @@ class ContactDiaryStoreV1: DiaryStoring, DiaryProviding {
 		guard case .success = updateDiaryResult else {
 			return nil
 		}
+		
+		migrateColumns(inTable: "ContactPerson")
+		migrateColumns(inTable: "Location")
 
 		registerToDidBecomeActiveNotification()
 	}
-
+	
+	func migrateColumns(inTable tableName: String) {
+		databaseQueue.inDatabase { database in
+			let queryResult = database.prepare("PRAGMA table_info(" + tableName + ")" )
```

Comment:
What's the deal with string subsitution?

```suggestion
			let queryResult = database.prepare("PRAGMA table_info(\(tableName))" )
```

Reply:
it is the same, i will rewrite it

Reply:
I will try the other way

Reply:
<img width="808" alt="Screenshot 2020-12-29 at 13 31 19" src="https://user-images.githubusercontent.com/15270737/103280960-4c7eab00-49da-11eb-8084-6a4547fe068f.png">
<img width="737" alt="Screenshot 2020-12-29 at 13 31 49" src="https://user-images.githubusercontent.com/15270737/103280966-50123200-49da-11eb-8b57-4d2187b6f8e6.png">

Reply:
as you see in the screenshots the .next() returns false with the normal String interpolation

Reply:
Thanks for the verification. I still wonder why this is the case 🤨